lib/FastImageSize.php

```diff
@@ -149,6 +149,7 @@ protected function getImagesizeUnknownType($filename)
 	 */
 	protected function getImageSizeByExtension($file, $extension)
 	{
+		$extension = strtolower($extension);
```

Comment:
This seems to be duplicating the strtolower() call that was introduced in the mentioned commit to the getImageSizeByExtension() method. We can of course move this to the getImageSizeByExtension() method which might actually make more sense. In that case the strtolower() call should probably be removed though: https://github.com/marc1706/fast-image-size/blob/master/lib/FastImageSize.php#L114